azion.config.cjs

```diff
@@ -243,19 +243,6 @@ const backRules = [
       rewrite: '/billing/graphql'
     }
   },
-  {
-    name: 'Route List Client Ids to Github',
```

Comment:
Might as well remove this Rule and Origin from azion/stage/azion.json and azion/prod/azion.json files right?

Reply:
I removed the settings